Kernel/Arch/aarch64/init.cpp

```diff
@@ -91,11 +92,15 @@ void __stack_chk_fail()
     Prekernel::halt();
 }
 
+using namespace Kernel;
+
 [[noreturn]] void __assertion_failed(char const* msg, char const* file, unsigned line, char const* func)
 {
     critical_dmesgln("ASSERTION FAILED: {}", msg);
     critical_dmesgln("{}:{} in {}", file, line, func);
 
+    // Used for printing a nice backtrace!
+    PANIC("Aborted");
     Prekernel::halt();
```

Comment:
Does PANIC() imply halt() already?

Reply:
Yes, you are correct! PANIC() indeed is defined as [[noreturn]] and in all cases will actually halt the processor.

I removed the Prekernel::halt().